docs/data/data-grid/selection/CellSelectionGrid.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import { DataGridPremium } from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+export default function CellSelectionGrid() {
+  const [rowSelection, setRowSelection] = React.useState(false);
+
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10,
+    maxColumns: 6,
+  });
+
+  return (
```

Comment:
For the Shift + Click range selection, is it possible to "remember" the first cell that started the range? 
Like in spreadsheet:

https://user-images.githubusercontent.com/13808724/200922650-ee9c591d-92ff-480c-833a-967570bcfed2.mov

Currently, the grid creates new range selection started from the last click cell:

https://user-images.githubusercontent.com/13808724/200922830-9a70d55b-afd6-4098-a8b5-5381f2e4e98c.mov

Reply:
We agreed on keeping the current behavior for now, as it requires some changes in how we manage the focused/outlined cell state.
@m4theushw will submit a follow-up PR to change the cell focus management so it's possible to always start a new range from a first selected cell of a previous range.
Did I capture it correctly?